packages/jest-cli/src/__tests__/watch.test.js

```diff
@@ -364,6 +364,139 @@ describe('Watch mode flows', () => {
     expect(run).toHaveBeenCalled();
   });
 
+  describe('when dealing with potential watch plugin key conflicts', () => {
+
```

Comment:
This line causes lint failure on CI

Reply:
Dang, should have checked. Just a sec.

Reply:
`yarn lint --fix` should solve your problem 👍

Reply:
It's odd, I guess my VS Code was overwhelmed with some extension, not to have auto-run Prettier on this.